src/Microsoft.Identity.Web/TokenAcquisition.cs

```diff
@@ -238,6 +237,32 @@ public class TokenAcquisition : ITokenAcquisition
             return accessToken;
         }
 
+        /// <summary>
+        /// Acquires a token from the authority configured in the app, for the confidential client itself (in the name of no user)
+        /// using the client credentials flow. See https://aka.ms/msal-net-client-credentials.
+        /// </summary>
+        /// <param name="scopes">scopes requested to access a protected API. For this flow (client credentials), the scopes
+        /// should be of the form "{ResourceIdUri/.default}" for instance <c>https://management.azure.net/.default</c> or, for Microsoft
+        /// Graph, <c>https://graph.microsoft.com/.default</c> as the requested scopes are defined statically with the application registration
+        /// in the portal, and cannot be overriden in the application.</param>
+        /// <returns>An access token for the app itself, based on its scopes</returns>
+        public async Task<string> AcquireTokenForAppAsync(IEnumerable<string> scopes)
+        {
+            if (scopes == null)
+            {
+                throw new ArgumentNullException(nameof(scopes));
+            }
+
+            // Use MSAL to get the right token to call the API
+            var application = GetOrBuildConfidentialClientApplication();
```

Comment:
There is already a class member named `application`. Please rename this class member to `_application` to avoid confusion.